setup.py

```diff
@@ -66,7 +66,7 @@ def run(self, *arg, **kw):
     'CherryPy<11.1',
     'click',
     'filelock',
-    'funcsigs ; python_version < \'3.5\'',
+    'funcsigs ; python_version < \'3\'',
```

Comment:
If I understand, this is kind of going out of our way to break compatibility with python between 3.0 and 3.4?

Reply:
Indeed, this is making an additional change (which I've structured as a separate final commit, in case we don't like it) to eliminate special cases around Python <3.5. However, if we don't support Python 3.4, it will only be a matter of time before something breaks under it, and going forward, this change somewhat simplifies some of the existing code in Girder.

@manthey I recall speaking to you about this, but I don't recall what your opinion was?